app/src/main/java/ch/epfl/qedit/model/Question.java

```diff
@@ -39,7 +48,10 @@ public Question(String title, String text, AnswerFormat format) {
         this.title = Objects.requireNonNull(title);
         this.text = Objects.requireNonNull(text);
         this.format = Objects.requireNonNull(format);
-        this.location = null;
+
+        // We store some dummy values, since nobody should try to access them anyway
+        this.longitude = 0;
```

Comment:
This might not be the cleanest way of doing it but I totally get why we do this. Ideally we might create to types of `Questions`...

Reply:
Yeah, I agree with you. Now there is only a boolean in the `Quiz`class to make the difference with treasure hunt quizzes. I could also put a boolean in the `Question` class, but it would be redundant, so that's why I didn't add it. But that's arguable, so let me know what you prefer.

Reply:
No, let's keep it that way.

Reply:
I agree, this is fine